tests/std/tests/P0896R4_ranges_test_machinery/test.cpp

```diff
@@ -26,9 +26,8 @@ constexpr bool iter_test() {
 
     STATIC_ASSERT(!movable<Element> || indirectly_writable<I, Element>);
 
-    constexpr bool can_write =
-        derived_from<Category,
-            output_iterator_tag> || (derived_from<Category, forward_iterator_tag> && assignable_from<Element&, Element>);
+    constexpr bool can_write = derived_from<Category, output_iterator_tag> //

```

Comment:
This is intentional (and mysterious-looking) - it forces clang-format to wrap the line. There are a few occurrences of this where clang-format can do something reasonable with a bit of assistance, without having to completely suppress it.

Reply:
I see...I was wondering if it was something like that!

Reply:
Yep: this is me forcing a reasonable line-break before the binary operator instead of allowing clang-format to break it in the middle of a perfectly good template-id. (Related: My belief that `requires` followed by `(` should be formatted differently in:
```c++
template <class T>
    requires (meow<T> + bar<T> < 42)
void f(T) { /* ... */ }
```
vs.
```c++
template <class T>
concept C =  requires(T x) { x + 42; );
```
has apparently delayed the clang-format concepts-and-requires-expressions patch enough to make it miss the Clang 11 fork. :sadpanda:)